force-app/main/default/classes/OpportunityTriggerHandler.cls

```diff
@@ -0,0 +1,78 @@
+public class OpportunityTriggerHandler {
+    // Handler methods
+    public static void beforeUpdate (List<Opportunity> newOpps) {
+        validateOpportunityAmmount(newOpps);
+        setPrimaryContact(newOpps);
+    }    
+    
+    public static void beforeDelete (List<Opportunity> oldOpps) {
+        preventDeletionOfClosedWonOppsForBanking(oldOpps);
+    }
+    
+    // Helper methods
+    private static void validateOpportunityAmmount (List<Opportunity> newOpps) {
+        /*
+        * Question 5
+        * Opportunity Trigger
+        * When an opportunity is updated validate that the amount is greater than 5000.
+        * Error Message: 'Opportunity amount must be greater than 5000'
+        * Trigger should only fire on update.
+        */
+        for (Opportunity opp : newOpps) {
+            if (opp.Amount < 5000) {
+                opp.addError('Opportunity amount must be greater than 5000');
```

Comment:
Try using top level static constant variables for the number 5000 and the string error.